core/arch/arm/kernel/thread.c

```diff
@@ -1075,6 +1075,15 @@ void thread_init_per_cpu(void)
 	set_abt_stack(l, GET_STACK(stack_abt[pos]));
 
 	thread_init_vbar(get_excp_vect());
+
+#ifdef CFG_TA_FTRACE_SUPPORT
+	/*
+	 * Enable accesses to frequency register and physical counter
+	 * register in el0 mode required for timestamping during
```

Comment:
Nitpicking: "el0 mode" should be "EL0/PL0". This is not a "mode" but an "exception" (ARMv8) or "privilege" (ARMv7) level.

Reply:
ok